bin/manala

```diff
@@ -15,8 +15,6 @@ require_once __DIR__.'/../vendor/autoload.php';
 use Manala\Command;
 use Symfony\Component\Console\Application;
 
-define('MANALA_DIR', \Phar::running() ?: __DIR__.'/../');
```

Comment:
Sure, that's only for convenient reasons (and preserve from any issue if the files requiring the get the project root are moved elsewhere).

But that's not a big deal actually :)